homeassistant/components/device_tracker/google_maps.py

```diff
@@ -60,25 +64,29 @@ def __init__(self, hass, config: ConfigType, see) -> None:
             self.success_init = True
 
         except InvalidUser:
-            _LOGGER.error('You have specified invalid login credentials')
+            _LOGGER.error("You have specified invalid login credentials")
             self.success_init = False
 
     def _update_info(self, now=None):
         for person in self.service.get_all_people():
-            dev_id = 'google_maps_{0}'.format(slugify(person.id))
+            try:
+                dev_id = 'google_maps_{0}'.format(person.id)
+            except TypeError:
+                _LOGGER.error("No location(s) shared with this account")
```

Comment:
It's possible to disable the location sharing temporary/easily? If yes, I would use the log level "warn" here.

Reply:
Yes, it's possible to disable it or to let it disable after a timeout.